ember-amount-input/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "ember-amount-input",
+  "version": "5.0.0",
+  "description": "The default blueprint for ember-cli addons.",
+  "keywords": [
+    "ember-addon"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/qonto/ember-amount-input.git"
+  },
+  "license": "MIT",
+  "author": "",
+  "files": [
+    "addon-main.cjs",
+    "dist"
+  ],
+  "scripts": {
+    "build": "rollup --config",
+    "lint": "concurrently 'npm:lint:*(!fix)' --names 'lint:'",
+    "lint:fix": "concurrently 'npm:lint:*:fix' --names 'fix:'",
+    "lint:hbs": "ember-template-lint . --no-error-on-unmatched-pattern",
+    "lint:js": "eslint . --cache",
+    "lint:hbs:fix": "ember-template-lint . --fix --no-error-on-unmatched-pattern",
+    "lint:js:fix": "eslint . --fix",
+    "start": "rollup --config --watch",
+    "test": "echo 'A v2 addon does not have tests, run tests in test-app'",
+    "prepack": "yarn run build",
+    "prepare": "yarn run build"
+  },
+  "dependencies": {
+    "@embroider/addon-shim": "^1.0.0"
```

Comment:
Should we keep `ember-cli-htmlbars` & `ember-cli-babel`? 

For babel I'm pretty sure no but just checking